weave-js/src/components/PagePanelComponents/Home/HomeFeaturedTemplates.tsx

```diff
@@ -0,0 +1,195 @@
+      <LayoutElements.HStack
+        style={{
+          gap: '16px',
+        }}>
+        {featuredTemplates.result.map((template: any, i: number) => (
+          <HomeFeaturedTemplateItem
+            key={i}
+            template={template}
+            setPreviewNode={setPreviewNode}
+          />
+        ))}
+      </LayoutElements.HStack>
+    </LayoutElements.VBlock>
+  );
+};
+
+const HomeFeaturedTemplateItem: React.FC<{
+  template: TemplateType;
+  setPreviewNode: SetPreviewNodeType;
+}> = ({template, setPreviewNode}) => {
+  const node = (
+    <HomePreviewSidebarTemplate
+      title={template.display_name}
+      setPreviewNode={setPreviewNode}
+      isTemplate={true}
+      actions={[]}>
+      <HomeFeaturedTemplateDrawer template={template} />
+    </HomePreviewSidebarTemplate>
+  );
+
+  return (
+    <LayoutElements.VStack
+      style={{
+        border: '1px solid #E5E5E5',
+        borderRadius: '8px',
+        padding: '16px',
+        gap: '8px',
+      }}>
+      <LayoutElements.HBlock
+        style={{
+          fontSize: '18px',
+          fontWeight: 800,
+        }}>
+        {template.display_name}
+      </LayoutElements.HBlock>
+      <LayoutElements.HStack>{template.description}</LayoutElements.HStack>
+      <LayoutElements.HBlock
+        style={{
+          marginTop: '12px',
+          justifyContent: 'flex-end',
+        }}>
+        <Button
+          variant="secondary"
+          size="medium"
+          onClick={() => {
+            setPreviewNode(node, '40%');
+          }}>
+          Use template
+        </Button>
+      </LayoutElements.HBlock>
+    </LayoutElements.VStack>
+  );
+};
+
+const TabContentWrapper = styled.div`
+  overflow-y: scroll;
+  padding: 16px;
+  height: 100%;
+  .markdown-body {
+    ul {
+      list-style: unset !important;
+    }
+  }
+`;
+TabContentWrapper.displayName = 'S.TabContentWrapper';
+
+const HomeFeaturedTemplateDrawer: React.FC<{
+  template: TemplateType;
+}> = ({template}) => {
+  const [tabValue, setTabValue] = React.useState('Instructions');
+  return (
+    <TWDrawerHack>
+      <LayoutElements.VBlock
+        style={{
+          height: '100%',
+          overflow: 'hidden',
+        }}>
+        <Tabs.Root
+          className="h-full"
+          value={tabValue}
+          onValueChange={(val: string) => setTabValue(val)}>
+          <Tabs.List className="px-16">
+            <Tabs.Trigger value="Instructions">1. Instructions</Tabs.Trigger>
+            <Tabs.Trigger value="Select Data">2. Select Data</Tabs.Trigger>
+          </Tabs.List>
+          <Tabs.Content value="Instructions" style={{height: '100%'}}>
+            <TabContentWrapper>
+              <ReactMarkdown
+                components={{
+                  // Ensures that links open in a new tab
+                  a: ({node, ...props}) => {
+                    return <TargetBlank {...props} />;
+                  },
+                  code({node, inline, className, children, ...props}) {
```

Comment:
This looks odd to me, should it be an arrow function?

Reply:
fixed